pkg/cluster/banzaicloud/types.go

```diff
@@ -78,21 +80,21 @@ type Taint string
 
 type AmazonProviderConfig struct {
 	AutoScalingGroup struct {
-		Name                    string  `json:"name" yaml:"name"`
-		Image                   string  `json:"image" yaml:"image"`
-		Zones                   Zones   `json:"zones" yaml:"zones"`
-		InstanceType            string  `json:"instanceType" yaml:"instanceType"`
-		LaunchConfigurationName string  `json:"launchConfigurationName" yaml:"launchConfigurationName"`
-		LaunchTemplate          string  `json:"launchTemplate" yaml:"launchTemplate"`
-		VPCID                   string  `json:"vpcID" yaml:"vpcID"`
-		SecurityGroupID         string  `json:"securityGroupID" yaml:"securityGroupID"`
-		Subnets                 Subnets `json:"subnets" yaml:"subnets"`
-		Tags                    Tags    `json:"tags" yaml:"tags"`
+		Name                    string  `json:"name" yaml:"name" binding:"required"`
+		Image                   string  `json:"image" yaml:"image" binding:"required"`
+		Zones                   Zones   `json:"zones" yaml:"zones" binding:"required"`
+		InstanceType            string  `json:"instanceType" yaml:"instanceType" binding:"required"`
+		LaunchConfigurationName string  `json:"launchConfigurationName" yaml:"launchConfigurationName" binding:"required"`
+		LaunchTemplate          string  `json:"launchTemplate" yaml:"launchTemplate" binding:"required"`
```

Comment:
LaunchTemplate is not used for now.
```suggestion
		LaunchTemplate          string  `json:"launchTemplate" yaml:"launchTemplate"`
```